torchvision/transforms/functional.py

```diff
@@ -446,22 +480,29 @@ def _get_perspective_coeffs(startpoints, endpoints):
     return res.squeeze_(1).tolist()
 
 
-def perspective(img, startpoints, endpoints, interpolation=Image.BICUBIC):
+def perspective(img, startpoints, endpoints, interpolation=Image.BICUBIC, fill=None):
     """Perform perspective transform of the given PIL Image.
 
     Args:
         img (PIL Image): Image to be transformed.
         startpoints: List containing [top-left, top-right, bottom-right, bottom-left] of the orignal image
         endpoints: List containing [top-left, top-right, bottom-right, bottom-left] of the transformed image
         interpolation: Default- Image.BICUBIC
+        fill (n-tuple or int or float or None): Pixel fill value for area outside the rotated
+            image. If None, defaults to 0. If int or float, the value is used for all bands respectively.
+            This option is only available for ``pillow>=5.0.0``.
+
     Returns:
         PIL Image:  Perspectively transformed Image.
     """
+
     if not _is_pil_image(img):
         raise TypeError('img should be PIL Image. Got {}'.format(type(img)))
 
+    opts = _parse_fill(fill, len(img.getbands()), '5.0.0')
```

Comment:
Nit: should we move `len(img.getbands()` into `_parse_fill()` and simply pass `img`? Otherwise by looking at this call it might not be clear that this represents the number of bands.

Reply:
Yep. Simply passing ``img`` is cleaner.